docs/modules/secret-operator/pages/cert-manager.adoc

```diff
@@ -0,0 +1,82 @@
+= Cert-Manager Integration
+
+https://cert-manager.io/[Cert-Manager] is a common tool to manage certificates in Kubernetes, especially when backed by an external
+Certificate Authority (CA) such as https://letsencrypt.org/[Let\'s Encrypt].
+
+The Stackable Secret Operator does not currently support managing Cert-Manager certificates directly, but it can be configured to consume certificates generated by it.
+
+[#caveats]
+== Caveats
+
+Cert-Manager is designed to manage relatively long-lived certificates that are stored in Kubernetes Secrets. By contrast,
+the Stackable Secret Operator is designed to generate temporary short-lived certificates.
+
+This has a couple of repercussions:
+
+- Longer-lived certificates mean that a leaked certificate has potential to be abused for longer.
+- Application teams may have access to read Secrets in their respective applications' Namespaces.
+
+Where possible, we recommend using the xref:secretclass.adoc#backend-autotls[`autoTls` backend] instead.
+
+[#issuer]
+== Configuring Cert-Manager
+
+NOTE: We recommend using the xref:secretclass.adoc#backend-autotls[`autoTls` backend] instead for self-signed PKIs. We use Cert-Manager's CA issuer here to show the broader concepts.
+
+To do this, we'll first need to teach Cert-Manager how we want it to create our certificates.
+
+In a production setup this will likely use an external CA such as ACME or OpenBao/Vault. However, to make this guide self-contained we will let Cert-Manager create
+a self-signed CA certificate instead.
```

Comment:
As per the [Documentation style guide][doc-style], we should write in first person.

```suggestion
To do this, you will first need to teach Cert-Manager how to create your certificates.

In a production setup this will likely use an external CA such as ACME or OpenBao/Vault. However, to make this guide self-contained, Cert-Manager will create
a self-signed CA certificate instead.
```



[doc-style]: https://docs.stackable.tech/home/stable/contributor/docs-style-guide#_tone_and_writing_style_google_developer_documentation_style_guide